cdc/log.cc

```diff
@@ -974,11 +976,12 @@ cdc::cdc_service::impl::augment_mutation_call(lowres_clock::time_point timeout,
         return make_ready_future<std::tuple<std::vector<mutation>, cdc::result_callback>>(std::make_tuple(std::move(mutations), result_callback{}));
     }
 
+    tracing::trace(tr_state, "CDC: Started generating mutations for log rows");
     mutations.reserve(2 * mutations.size());
 
     return do_with(std::move(mutations), service::query_state(service::client_state::for_internal_calls(), empty_service_permit()),
-            [this, timeout, i] (std::vector<mutation>& mutations, service::query_state& qs) {
-        return transform_mutations(mutations, 1, [this, &mutations, timeout, &qs] (int idx) {
+            [this, timeout, i, tr_state = std::move(tr_state)] (std::vector<mutation>& mutations, service::query_state& qs) {
```

Comment:
Missing `mutable`. 
As a result `tr_trace` (together with all other captured by-value variable) is going to be `const` inside this lambda and all further `std::move()s` on it are going to be copies.